skills/plane-backlog/scripts/plane_create_issue.py

```
@@ -0,0 +1,500 @@
+    try:
+        req = urllib.request.Request(url, data=json.dumps(payload).encode("utf-8"), headers=headers, method="POST")
+        with urllib.request.urlopen(req) as resp:
+            data = json.loads(resp.read().decode("utf-8"))
+            issue_id = data.get("id")
+            seq_id = data.get("sequence_id")
+            issue_url = f"{plane_host}/{workspace_slug}/projects/{prj_id}/issues/{issue_id}"
+            
+            if is_intake:
+                intake_url = f"{plane_host}/api/v1/workspaces/{workspace_slug}/projects/{prj_id}/intake-issues/"
+                try:
+                    intake_req = urllib.request.Request(intake_url, data=json.dumps({"issue": issue_id}).encode("utf-8"), headers=headers, method="POST")
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)